scripts/db_migrator.py

```diff
@@ -725,6 +742,8 @@ def common_migration_ops(self):
             self.migrate_mgmt_ports_on_s6100()
         else:
             log.log_notice("Asic Type: {}, Hwsku: {}".format(self.asic_type, self.hwsku))
+        
+        self.migrate_port_qos_map_global()
```

Comment:
why is this in the common migration path? I thought it should be based on a version.

Reply:
Updated. Please take a look. Thanks